cmd/config.go

```diff
@@ -56,13 +58,16 @@ func newConfigCmd() *cobra.Command {
 				return err
 			}
 
-			return listConfig(stack, showSecrets)
+			return listConfig(stack, showSecrets, jsonOut)
 		}),
 	}
 
 	cmd.Flags().BoolVar(
 		&showSecrets, "show-secrets", false,
 		"Show secret values when listing config instead of displaying blinded values")
+	cmd.Flags().BoolVarP(
+		&jsonOut, "json", "j", false,
+		"Emit output as JSON")
```

Comment:
Super nitpick (sorry): I'm OK with `"Emit output as JSON"`, but just wanted to point out that we're slightly inconsistent with the wording used in other commands.

 - `pulumi stack output` uses `"Emit outputs as JSON"`, which makes sense because it's all about "outputs". I believe this was the first command that had support for emitting as JSON.
- `pulumi logs` and `pulumi stack ls` use `"Emit outputs as JSON"`, which looks like it was just copy/pasted from `pulumi stack output`.
- `pulumi stack tag` I chose to use `"Emit stack tags as JSON"`.

It might be worth standardizing on `"Emit output as JSON"` across the board, or using the appropriate noun for each type of thing that is being outputted, e.g. `"Emit config as JSON"`, `"Emit logs as JSON"`, `"Emit stacks as JSON"`, etc.

Edit: I see you updated it for `pulumi stack ls`.

Reply:
> It might be worth standardizing on "Emit output as JSON" across the board, 

This makes sense, will do.